epochStart/bootstrap/epochStartMetaBlockProcessor.go

```diff
@@ -63,10 +65,10 @@ func NewEpochStartMetaBlockProcessor(
 	if !(consensusPercentage > 0 && consensusPercentage <= 100) {
 		return nil, epochStart.ErrInvalidConsensusThreshold
 	}
-	if minNumConnectedPeers < 1 {
+	if minNumConnectedPeers < minNumConnectedPeers {
```

Comment:
Could you rename with caps M the constant? They have the same name in left and right side of this compare

Reply:
you're right. good find. renamed the received parameter and kept the constant name the same